source/app/utils/redaction.js

```diff
@@ -0,0 +1,29 @@
+import uuid from 'uuid/v4';
+
+export const normalizeRedactionResponse = (redactions) => {
+  const normalizedRedactions = {};
+
+  redactions.forEach((redaction) => {
+    if (!normalizedRedactions[redaction.page]) normalizedRedactions[redaction.page] = {};
+
+    normalizedRedactions[redaction.page][uuid()] = {
+      Top: redaction.top,
+      Left: redaction.left,
+      Width: redaction.width,
+      Height: redaction.height,
+    };
+  });
+
+  return normalizedRedactions;
+};
+
+export const getRedactionsDtoFromRedactions = (redactions) =>
```

Comment:
getRedactionsDtoFromRedactions - from the name, what this function does is unclear to me. Can we change it?

Reply:
Renamed to `getRedactionsDto` per offline discussion

Reply:
NIT: Can we just have it as RedactionDTO?